.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug Report
+about: Create a report to help us improve NovelRT
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**_Note: for support questions, please use the #engine-user-help channel in our [Discord](https://discord.com/invite/V7SgCkm) or [create a discussion](https://github.com/novelrt/NovelRT/discussions/new). This repository's issues are reserved for feature requests and bug reports._**
```

Comment:
```suggestion
**_Note: for support questions, please use the #engine-user-help channel in our [Discord](https://discord.novelrt.dev/) or [create a discussion](https://github.com/novelrt/NovelRT/discussions/new). This repository's issues are reserved for feature requests and bug reports._**
```

Reply:
I forgot we had this loooool (I was trying to find it and couldn't)